sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -33,6 +33,10 @@
 #include "modules/computer_vision/viewvideo.h"
 #include "modules/computer_vision/cv.h"
 
+#include "modules/computer_vision/video_thread.h"
+
+#include "boards/bebop.h"
```

Comment:
The source files are defined in the makefiles in conf/boards. But I think you also have to add the headers correctly here.